src/server.ts

```diff
@@ -20,22 +20,24 @@ router.get('/rovers/:roverName/photos/:cameraType', async (req:any, res:any) =>
     let roverPhotoList;
     try {
         roverPhotoList = await requestHelper.getRoverPhotos(req.params.roverName, req.params.cameraType);
+        console.log(roverPhotoList);
     } catch (error) {
         console.log(error);
         res.sendStatus(400);
-    } finally {
-        if (roverPhotoList != null) {
-            let roverPhotoImages: string [] = [];
-            roverPhotoList.photos.map((photo) => {
-                roverPhotoImages.push(photo.img_src);
-            })
-            console.log(roverPhotoImages);
-            roverPhotoImages.forEach((photo) => {
-
-            })
+    }
+
+    if (roverPhotoList != null) {

```

Comment:
You'd probably need an `else` clause for this `if`. (What if the list is `null`?)